R/qview.R

```diff
@@ -32,6 +34,29 @@ cancel_button_creator <- function(index, .ns) {
   )
 }
 
+#' Function to grab a dataframe of the jobs in the queue for the given user.
+#' @param queue_obj The RSimpleQueue object currently in use.
+#' @param .user THe user as a character string -- only this user's jobs will be returned.
+#' @importFrom dplyr filter arrange desc
+#' @importFrom purrr is_null
+get_queue_data <- function(queue_obj, .user) {
+  return_value <- NULL
+
+  jobs_df <- jobs_to_df(
+    queue_obj$get_jobs()
+  )
+
+  if(!purrr::is_null(jobs_df)) {
+    filtered_df <-
+      dplyr::arrange(jobs_df, dplyr::desc(id)) %>%
+      dplyr::filter(user == .user)
+    if(nrow(filtered_df) > 0) {
```

Comment:
Handling edge-case where jobs_df has jobs, but none of them apply to the current user.